pkg/omf/cli/omf.install_package.fish

```diff
@@ -1,5 +1,12 @@
-function omf.install_package
+# Return success for at least one successful install.
+function omf.install_package 
+  set -l return_code $OMF_UNKNOWN_ERR
   for search in $argv
     omf.install --pkg $search
+    set -l install_code $status
+    if test $install_code -eq 0
```

Comment:
Yep, we are not really worried about naming collision as the prefix are recommended/mandatory.